src/app/notification/examples/notification-service-observer-example.component.ts

```diff
@@ -0,0 +1,51 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    OnInit,
+    ViewEncapsulation
+  } from '@angular/core';
+import { Observable } from 'rxjs';
+
+  import { Notification } from '../notification';
+  import { NotificationService } from '../notification.service';
+  import { NotificationType } from '../notification-type';
+
+  @Component({
+    encapsulation: ViewEncapsulation.None,
+    selector: 'notification-service-observer-example',
+    templateUrl: './notification-service-observer-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+  })
+  export class NotificationServiceObserverExampleComponent implements OnInit {
+    notifications: Observable<Notification[]>;
+
+    constructor(private notificationService: NotificationService) {
+      notificationService.setDelay(8000);
+    }
+
+    ngOnInit(): void {
+      this.notifications = this.notificationService.getNotificationsObserver;
+      // Display notifications to illustrate how
+      // one can react to a notifications observer
+      const exampleTimer = setInterval(_ => {
+        this.notificationService.message(
+          NotificationType.SUCCESS,
+          'Example header',
+          'Open js console to see notifications observer in action',
+          false,
+          null,
+          null
+        );
+      }, 1000);
+
+      setTimeout(_ => {
+        clearInterval(exampleTimer);
+      }, 6000);
```

Comment:
You might consider replacing the timer with a button? It may be a cleaner way to demo this example without a building up messages while switching tabs. At least you won't generate more messages than you can click.

On another note, the basic and observer examples should be able to use the same notification service. Even if there are a build up of messages, I expect the basic example would still show those messages, right?